.gitignore

```diff
@@ -47,7 +47,11 @@ stamp-h1
 test-driver
 ylwrap
 
+# quilt, tool for patches
+.pc
```

Comment:
Please make it `.pc/` - so that pkgconfig files are NOT blocked, only quilt's .pc directory

Reply:
Done